src/main/java/socialite/logic/parser/CliSyntax.java

```diff
@@ -12,4 +12,7 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    /* SociaLite specific prefix definitions */
```

Comment:
I think this comment is not that meaningful since everything here is SociaLite anyways. Maybe something like "Social media handle prefix definitions"?

Reply:
Yeap that makes much more sense, will be fixing that in another commit.